packages/@icon-magic/config-reader/src/config.ts

```diff
@@ -0,0 +1,102 @@
+import * as path from 'path';
+import * as glob from 'glob';
+import { loadConfigFile } from './config-loader';
+import { validateConfigSchema } from './config-validator';
+import { Icon } from '@icon-magic/icon-models';
+import { IconSetHashInterface } from '@icon-magic/icon-models/src/interface';
+import { isDirectory } from '@icon-magic/icon-models/dist/src/utils/files';
+
+export class Config {
```

Comment:
Describe `Config`.

Reply:
Also, please add comment docs for all methods on this class 👍